plover/assets/main.json

```diff
@@ -28830,7 +28694,6 @@
 "HRAPS/-G": "lapsing",
 "HRAPS/-S": "lapses",
 "HRAPZ": "laps",
-"HRAR": "{^ular}",
```

Comment:
I think this is not on purpose

Reply:
I've no idea how that happened.  I don't show that I deleted it, yet its gone.  Will add it back.